Python/crossfire/tests/test_occurences.py

```diff
@@ -57,7 +57,49 @@ def test_occurrences_with_obligtory_and_id_cities_parameters():
     client = Mock()
     client.get.return_value = dummy_response()
     client.URL = "https://127.0.0.1/"
-    tuple(Occurrences(client, id_state="42", id_cities="21", limit=2))
+    tuple(Occurrences(client, id_state="42", id_cities="21", limit=1))
     client.get.assert_called_once_with(
-        f"{client.URL}/occurrences?idState=42&idCities=21&page=1"
+        f"{client.URL}/occurrences?idState=42&page=1&idCities=21"
+    )
+
+
+def test_occurrences_with_obligtory_and_two_id_cities_parameters():
+    client = Mock()
+    client.get.return_value = dummy_response(True)
+    client.URL = "https://127.0.0.1/"
+    tuple(
+        occurence
+        for occurence in Occurrences(
+            client, id_state="42", id_cities=["21", "11"], limit=1
+        )
+    )
```

Comment:
Você tentou simplificar apenas convertendo o gerador em tupla?

```suggestion
    tuple(Occurrences(client, id_state="42", id_cities=["21", "11"], limit=1))
```